strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ArtifactResolutionServiceImpl.java

```diff
@@ -130,5 +138,49 @@ public URL resolveArtifactResource(String storageId,
                                    .resolve(artifactResource)
                                    .toURL();
     }
+
+    @Override
+    public RepositoryFileAttributes getAttributes(String storageId,
+                                                  String repositoryId,
+                                                  String artifactPath) throws IOException, NoSuchAlgorithmException, ArtifactTransportException, ProviderImplementationException
+    {
+        logger.debug("******");
+        
+        artifactOperationsValidator.validate(storageId, repositoryId, artifactPath);
+
+        final Repository repository = getStorage(storageId).getRepository(repositoryId);
+     
+        LayoutProvider<?> layoutProvider = layoutProviderRegistry.getProvider(repository.getLayout());
+        RepositoryProvider repositoryProvider = repositoryProviderRegistry.getProvider(repository.getType());
+        
+        logger.debug("REpository TYpe = "+repositoryProvider.getAlias());
+        
+        RepositoryPath path = null;
+        
+        try
+        {
+            if(!repositoryProvider.getAlias().equals("group"))
+                path = layoutProvider.resolve(repository).resolve(artifactPath);
+            else
+                path = ((GroupRepositoryProvider)repositoryProvider).getPath(storageId,repositoryId,artifactPath);
```

Comment:
* Same as above.
* Also, please always use spaces after `+` and `,`, as this is the code style of the project. Thanks! :)